tests/validate/gometalinter.sh

```diff
@@ -11,6 +11,7 @@ exec gometalinter.v1 \
 	--enable-gc \
 	--exclude='error return value not checked.*(Close|Log|Print).*\(errcheck\)$' \
 	--exclude='.*_test\.go:.*error return value not checked.*\(errcheck\)$' \
+	--exclude='declaration of.*err.*shadows declaration.*\(vetshadow\)$'\
```

Comment:
Is this part of a different PR?

Reply:
I don't recall this being there when I reviewed earlier, a merge hiccup perhaps?

Reply:
It was failing on testing for some reason.  Lint was checking on vetshadow of err.  
BTW I am seeing no speed increase by adding this patch.  So I think we should ignore it and add the vetshadow patch separately.

Reply:
You wouldn't see a speed improvement the first time, since `-i` is more or less doing a `go install`, saving compiled packages to disk so that they don't have to be recompiled the second time -- compare by running `go build -v -i -o buildah ./cmd/buildah` twice.

Reply:
Ok I will reopen.
The question @baude had was whether a make clean would reverse the go install code so everything would be rechecked.

Reply:
It doesn't, but adding a `go clean -i -r ./cmd/buildah` does.  I'm not really a fan of installing things under $GOPATH as a side-effect of compiling our binaries, though.